index.js

```diff
@@ -0,0 +1,77 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const fs = require('fs');
+const filePath = core.getInput('path')
+const { request } = require('@octokit/request');
+const { Octokit } = require("@octokit/core");
+fs.readdir('./', (err, files) => {
```

Comment:
There should be a function that gets called to do the file checks.
For every requirement, try having a separate function